extensions/objects/templates/math-expression-content-editor.directive.ts

```diff
@@ -35,15 +39,58 @@ angular.module('oppia').directive('mathExpressionContentEditor', [
       controllerAs: '$ctrl',
       controller: ['$scope', function($scope) {
         var ctrl = this;
+        var convertLatexStringToSvg = function(inputLatexString) {
+          var emptyDiv = document.createElement('div');
+          var outputElement = angular.element(emptyDiv);
+          var $script = angular.element(
+            '<script type="math/tex">'
```

Comment:
Why do we need a script tag here, add code comment? (Do we need a closing tag? not sure!)

Reply:
I've added a comment explaining the reason